create-pr.sh

```diff
@@ -0,0 +1,157 @@
+#!/bin/bash
+
+# create-pr.sh - Automated PR creation script for django-ace
+# Usage: ./create-pr.sh [branch-name] [pr-title] [commit-message]
+
+set -e  # Exit on any error
+
+# Colors for output
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[1;33m'
+BLUE='\033[0;34m'
+NC='\033[0m' # No Color
+
+# Function to print colored output
+print_status() {
+    echo -e "${BLUE}[INFO]${NC} $1"
+}
+
+print_success() {
+    echo -e "${GREEN}[SUCCESS]${NC} $1"
+}
+
+print_warning() {
+    echo -e "${YELLOW}[WARNING]${NC} $1"
+}
+
+print_error() {
+    echo -e "${RED}[ERROR]${NC} $1"
+}
+
+# Check if GitHub CLI is installed
+if ! command -v gh &> /dev/null; then
+    print_error "GitHub CLI (gh) is not installed. Install it with: brew install gh"
```

Comment:
I'm not a fan of the gh command being mandatory, I'll lead to people thinking it's needed to open a PR, while it's not (just click the link after pushing, don't need a whole program for that).